verticapy/datasets/generators.py

```diff
@@ -1,18 +1,19 @@
 """
-(c)  Copyright  [2018-2023]  OpenText  or one of its
-affiliates.  Licensed  under  the   Apache  License,
-Version 2.0 (the  "License"); You  may  not use this
-file except in compliance with the License.
-
-You may obtain a copy of the License at:
-http://www.apache.org/licenses/LICENSE-2.0
-
-Unless  required  by applicable  law or  agreed to in
-writing, software  distributed  under the  License is
-distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
-OR CONDITIONS OF ANY KIND, either express or implied.
-See the  License for the specific  language governing
-permissions and limitations under the License.
+..
+    (c)  Copyright  [2018-2023]  OpenText  or one of its
```

Comment:
@tconsigny  why did you indent the copyright header?

@sitingren is it possible to include this header in the license instead of having it in all our files? It makes everything harder to parse...

Reply:
@tconsigny I don't think there is a need to comment it out. The way how I have structured the main auto-doc file, it will not matter.